viper/parser.py

```diff
@@ -645,6 +647,8 @@ def parse_expr(expr, context):
             new_unit = combine_units(left.typ.unit, right.typ.unit, div=True)
             if rtyp == 'num':
                 o = LLLnode.from_list(['sdiv', left, ['clamp_nonzero', right]], typ=BaseType(ltyp, new_unit), pos=getpos(expr))
+            elif ltyp == rtyp == 'num256':
+                o = LLLnode.from_list(['div', left, right], typ=BaseType('num256', new_unit), pos=getpos(expr))
```

Comment:
There's a potential for data loss here. I'm not sure if there's a way to avoid it (such as above for `num`, which also has potential data loss). Discuss.

Reply:
And by data loss, I mean decimal point flooring (or does it round? @DavidKnott )